README.md

```diff
@@ -1,77 +1,70 @@
-  In order to find out if your code works not just on your machine but in all environments, check the build status. 
-  In PR you will see Azure Pipelines and it should have x check passed. 
-  Todo: screen shot
-  ### Mentor code review
-  In order for your PR to be complete, it needs a code review of at least 2 mentors. 
-  Both of the mentors will have to approve your code, otherwise you won't have a PR completed. 
-  How to do code review:
-  As a reviewer- Todo: link
-  As a reviewee-Todo: link
-  Code reviews on GitHub- Todo: link
-  
-  ### Finishing homework
-  You can consider your homework complete if all of the above steps pass. 
-  **Don't merge the PR to master branch**
-  A single best homework implementation will be merged to master. 
-
-  
+# Homework 3
 
-  # Community
-  If you want to join as either mentor or pupil or have any questions, please refer to our discord Community.  
-  https://discord.gg/rCMKcUU
-  
-  
-
-
-
- 
-
- 
+## Hints
+* This will make homework 4 much more easy:
+  * Do use Environment.NewLine if you want to concanate a string with a newline command (or use WriteLine).
+  * Do use [CultureInfo.InvariantCulture](https://docs.microsoft.com/de-de/dotnet/api/system.globalization.cultureinfo.invariantculture?view=netcore-3.1) when reading in float numbers. (Due to differences in parsing numbers with '.' and ',' character.
+* The better your design, the easier it will be to find and fix errors!
+* Sometimes you have to sync new changes from the C# Inn repository:
+  * Click the pencil symbol at the bottom right.
+  * Commit your changes or stash them (after finishing homework 2 there should none be left).
+  * Choose the correct branch in the bottom right (in this case Chapter1/Homework/3 should already be selected).
+  * Click the house button straight above (upper end).
+  * Click the sync button.
+  * Click fetch.
+  * Click pull.
 
+## Task
+1) Open the Homework 3:
+  * Choose the Chapter1/Homework/3 branch in the bottom right.
+    * If it is not there already:
+      * Click manage branches.
+      * Open remotes/origin/Chapter1/Homework and rightclick 3 and choose fetch.
+      * There should be a "Chapter1/Homework/3" path be available now. Choose that one.
+    * If it was there already you should sync as in the hint above.
+  * Open the solution explorer and Bootcamp.sln (if not already open).
+2) Take homework 2 code, place it in the Program.cs file of the current solution and make sure it still compiles and runs.
+3) Create a file Lesson3.cs:
+  * Rightclick BootCamp.Chapter.
+  * Click add.
+  * Click class.
+  * Type in Lesson3 and hit enter. (You can omit the .cs, it will be put there anyway.)
+4) Write a public static void Demo() function in this file.
+5) Put all Program.cs logic to Lesson3.cs. Call it from Main function.
+  * Do not delete the function or it's curly braces.
+  * Write between the curly braces "Lesson3.Demo();" without the ".
+  * This should look like this:
+  ```csharp
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, add 4 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896222)